frontend/app/page.tsx

```diff
@@ -88,6 +123,7 @@ export default function SignUp() {
                   label="Last Name"
                   name="lastName"
                   autoComplete="family-name"
+                  onChange={(event) => setLast(event.target.value)}
```

Comment:
**question**: This is a generalized question to discuss in a group but is there benefit to our application from having a last name component? I know that most websites do have last name as a parameter, so if we are mocking it then it may make sense. But I have never understood why applications need a last name and it seems like it adds to the lack of data control for the user.